src/event.rs

```diff
@@ -253,6 +253,25 @@ pub enum Event<T: 'static> {
     ///
     /// - **macOS / Wayland / Windows / Orbital:** Unsupported.
     MemoryWarning,
+
+    /// Emitted when the application has received a reopen request from OS.
+    ///
+    /// ## Platform-specific
+    ///
+    /// ### macOS
+    ///
+    /// On macOS, the `Reopen` event is emitted in response to an [`applicationShouldHandleReopen`]
+    /// callback, which is typically called when the user clicks on the application's icon in the Dock.
```

Comment:
Copied more precise text from the docs.

```suggestion
    /// callback, which is usually called whenever the Finder reactivates an already running
    /// application because the user double-clicked it again or used the dock to activate it.
```